gensim/models/lsimodel.py

```diff
@@ -348,6 +348,15 @@ def add_documents(self, corpus, chunksize=None, decay=None):
         """
         logger.info("updating model with new documents")
 
+        if isinstance(corpus, list):
+            logger.warning(
+                "taking plain list as a corpus for LSI;"
```

Comment:
I also think that the logging messages formatted as proper English sentences look better. I have just seen that the rest of the code in this file is always starting the logging message with lower case letter and ends with semicolon and I thought that it's the "gensim code style".

Fixed the problem with words sticking together.